libs/entry-components/configure-theming.md

```diff
@@ -0,0 +1,435 @@
+
+| Submap       | Description                                                             | Possible Options                      |
+|--------------|-------------------------------------------------------------------------|----------------------------------------|
+| **general**  | Influence the overall appearance and behavior of components. | `typography`, `fonts`, `spacing`, `inputs`, `buttons` |
+| **tables**   | Customize table component styling.                                       | `cells`, `rows`, `contents`            |
+| **dialogs**  | Set dialog font size.                                                   | `title`                                |
+
+```scss
+$custom-theme: (
+ general: (
+ // General configuration options go here
+ ),
+ tables: (
+ // Table-related configuration options go here
+ ),
+ dialogs: (
+ // Dialog-related configuration options go here
+ )
+);
+```
+
+### General Configuration
+
+The `general` section in the `$custom-theme` configuration contains settings that shape the overall look and behavior of the components.
+
+#### Density
+
+The density property typically accepts values like **-1**, **0** (default), and **1**, which correspond to different levels of density. For  [more info check](https://m2.material.io/design/layout/applying-density.html#usage):
+
+- `-1`: Reduces the spacing and makes elements more compact.
+- `0`: Default density according to the Material Design guidelines.
+- `1`: Increases spacing, providing a more relaxed and open layout.
+
+```scss
+$custom-theme: (
+ general: (
+ density: 0
+ )
+);
+```
+
+#### Colors
+
+Defines various color properties:
+
+- `primary`: Main color used in application, it sets the tone for the overall theme
+- `accent`: Highlight certain elements and create visual interest
+- `font`: Specifies the default font color.
+- `disabled`: Style elements that are in a disabled state:
+  - `foreground`: Text color for disabled elements
+  - `background`: Background color for disabled elements
+
+```scss
+$custom-theme: (
+ general: (
+  density: 0,
+  colors: (
+   primary: #2581C4,
+   accent: #EA518D,
+   font: #323232,
+   disabled: (
+    foreground: rgb(0 0 0 / .38),
+    background: rgb(0 0 0 / .12)
+   )
+  )
+ )
+);
+```
+
+#### Fonts
+
+Allow customization of typography and font styles based od Angular Material typography levels. Each font related property in configuration can have **family** and **size** values.
+
+- `hero-titles`: Define typography for h1, h2, h3, h4 elements.
+- `titles`: Typography h3, h4 elements
+- `body`: Define typography for base body text.
+- `buttons`: Typography for buttons and anchors
+
+> **_NOTE:_**  Don't forget to check if you [corrected configure](#importing-fonts) or import font family's
```

Comment:
Please rephrase: Don't forget to check whether font is being applied by checking font families etc.